MyString.java

```diff
@@ -18,13 +18,36 @@ public static void main(String[] args) {
 
     /** Returns the lowercase version of the given string. */
     public static String lowerCase(String str) {
-        // Replace the following statement with your code
-        return null;
+        String Str = "";
+        
+        for (int i = 0; i < str.length(); i++) {
+            if ((str.charAt(i) >= 65) && (str.charAt(i) <= 90)) {
+              char a = (char) (str.charAt(i) + 32);
```

Comment:
1. use char values not ASCII values
2. `+ 32` is not clear use `(char)(str.charAt(i) - 'A' + 'a')`